Simplenote/src/main/res/values/strings.xml

```diff
@@ -113,10 +113,10 @@
 
     <!-- Tags -->
     <string name="add_tag">Add Tag</string>
-    <string name="dialog_tag_add_button_negative">@android:string/cancel</string>
-    <string name="dialog_tag_add_button_positive">@string/save</string>
-    <string name="dialog_tag_add_hint">@string/tag_name</string>
-    <string name="dialog_tag_add_title">@string/add_tag</string>
+    <string name="dialog_tag_add_button_negative" translatable="false">@android:string/cancel</string>
+    <string name="dialog_tag_add_button_positive" translatable="false">@string/save</string>
+    <string name="dialog_tag_add_hint" translatable="false">@string/tag_name</string>
+    <string name="dialog_tag_add_title" translatable="false">@string/add_tag</string>
     <string name="dialog_tag_error_message">An error occurred adding the tag. Please, try again. If the problem continues, contact us at %1$s%2$s%3$s%4$s%5$s for help.</string>
     <string name="dialog_tag_error_message_email" translatable="false">\u003c\u0061\u0020\u0068\u0072\u0065\u0066\u003d\u0022\u006d\u0061\u0069\u006c\u0074\u006f\u003a\u0073\u0075\u0070\u0070\u006f\u0072\u0074\u0040\u0073\u0069\u006d\u0070\u006c\u0065\u006e\u006f\u0074\u0065\u002e\u0063\u006f\u006d\u0022\u003e\u0073\u0075\u0070\u0070\u006f\u0072\u0074\u0040\u0073\u0069\u006d\u0070\u006c\u0065\u006e\u006f\u0074\u0065\u002e\u0063\u006f\u006d\u003c\u002f\u0061\u003e</string>
```

Comment:
I'm not sure what that is (didn't took the time to decode the hex entities to decipher the message, I'm on my phone so not easy ^^) and why this is escaped like that. If that's supposed to be an actual text once hex-decoded, not sure why it would be untranslated anyway… if there's a valid reason to all that, maybe worth an XML comment in there? 🤔

Reply:
Now that I'm back on my Mac, I checked and the hex, once decoded, translates to this (well I've just replaced a couple of chars with `_` in my translation to avoid crawling – see below – but you can guess what they actually are):
```
<a href="m_ilto:su_port_a_simple_ote_com">su_port_a_simple_ote_com</a>
```

* I can now understand why this would be `translatable="false` (unrelated to the other cases with `@string/…`, just because an email address doesn't need translation)
* But I'm still not clear why this was all in hex rather than be `&lt;a href="…"&gt;…&lt;/a&gt;`.

My guess is this was done to obfuscate the email address from robots that would try to crawl the OSS repo for email addresses (which is why I've obfuscated it in this very comment too, just in case), especially after the attack we had a while ago (for account creation, but still). But even if that's the reason, which would indeed justify obfuscating with hex, I think we could at least benefit from an XML comment 🙂